readme.md

```
@@ -303,6 +305,10 @@ After any upstream sync, verify:
 
 ---
 
+## Code Quality
+
+<img src="scorecard.png" width="100%">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)